res/CircleSchema/0.6/circle_schema.fbs

```diff
@@ -84,8 +84,12 @@ table QuantizationParameters {
   // Given a quantized value q, the corresponding float value f should be:
   //   f = scale * (q - zero_point)
   // For other quantization types, the QuantizationDetails below is used.
-  min:[float];  // For importing back into tensorflow.
-  max:[float];  // For importing back into tensorflow.
+  // NOTE min/max values are valid if
+  // 1. length of min/max == 0 or
+  // 2. length of min/max == length of scale/zero_point
+  // Otherwise, min/max are not valid (undefined behavior).
+  min:[float];
+  max:[float];
```

Comment:
I'm not sure about the versioning but is it OK to go with same `0.6` with these changes?

Reply:
@jinevening , have you tests with files generated before-after these attributes?

Q) why not add at the end?

Reply:
Ah, this PR just updates comments (about min/max). So I think we don't have to increase version number.

Reply:
Ah, I missed "before" codes ... 😭